deploy/olm-catalog/service-telemetry-operator/Dockerfile.in

```diff
@@ -13,7 +13,7 @@ LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
 LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v0.19.4
 LABEL operators.operatorframework.io.metrics.project_layout=ansible
 LABEL com.redhat.delivery.operator.bundle=true
-LABEL com.redhat.openshift.versions="=v4.10"
+LABEL com.redhat.openshift.versions="v4.10-v4.11"
```

Comment:
This is what controls which channels our Operator is published in when being imported. I'm not 100% sure if this is picked up by the OperatorHub controllers on OCP to limit the bundle installation during testing, but either way the change needs to happen. More just a curiosity as to how much this is referred to across the system.